a/ampifitia.md

```diff
@@ -0,0 +1,9 @@
+# Ampifitia
+
+![alt text](ampi.JPG)
```

Comment:
Please change the alt text to the artist name.
Please change the filename to the artist name.